### PR TITLE
Dog fooding the DatasetDataGrid

### PR DESCRIPTION
**Pull request by ramfox**
- Datagrid look doesn't really match the rest of the site
  - looking into the styling will also help me get familiar with how exactly the css is loaded, etc
- add bounds for height and width
- make columns resizeable

- **When you resize the window, your nicely resized columns go back to their default, see how to not make that happen**
- **Look into multiple columns with same name bug, see which to keep or how to prevent in the future**

### CODE REVIEW

**Comment by ramfox** [comment]
Added defaultColumnWidth!!
- perhaps there is a version where we use the max of column header or our best fit width, not sure if it's better to see more data or have clearer read on column names

File: app/utils/defaultColumnWidth.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+const defaultColumnWidth = (dataset, data) => {
+  const headers = initialObj(dataset)
+  return data.reduce(reducer, headers)
+}
+
+export default defaultColumnWidth
+// const result = columnWidth(data, datasetRef)
+
+// console.log(result)
```
Comment (b5) [inline app/utils/defaultColumnWidth.js:120]: Nice work! Let's set aside some time next week to review this, I have a few suggestions for improving legibility, mainly by breaking this down into component steps & introducing a map function.

File: app/components/DatasetDataGrid.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React, { PropTypes } from 'react'
 import ReactDataGrid from 'react-data-grid'
 import { datasetProps } from '../propTypes/datasetRefProps.js'
 import Spinner from '../components/Spinner'
+import defaultColumnWidth from '../utils/defaultColumnWidth'
```
Comment (b5) [inline app/components/DatasetDataGrid.js:5]: should be plural `defaultColumnWidths`

**Review by b5 — APPROVED** [review]
a few little things, but looks great! 🇲🇰 🛴

**Pull request merged**